chapter13/dtj/ex7

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Std;
+use File::Basename;
+
+our($opt_s);
+getopts('s');
+my $link_function = $opt_s ? \&CORE::symlink : \&CORE::link;
+die "Need 2 arguments, source and destination" unless @ARGV == 2;
```

Comment:
Does the ARGV check here work with the getopts earlier? i.e. does getops remove the -s from ARGV if its found?